frontend/src/metabase/actions/containers/ActionCreator/FormCreator/OptionEditor.tsx

```diff
@@ -34,7 +34,7 @@ export const OptionPopover = ({
     <TippyPopoverWithTrigger
       placement="bottom-end"
       triggerContent={
-        <Icon name="list" size={14} tooltip={t`Change options`} />
+        <Icon name="list" size={16} tooltip={t`Change options`} />
```

Comment:
I didn't want to block this from merging, but I'm voicing my observation here. Even though the 2 icons have the same size, visually they're different.

## Both 16
<img width="66" alt="Screenshot 2023-03-03 at 7 04 14 PM" src="https://user-images.githubusercontent.com/1937582/222805371-b9770dec-f223-45a6-b558-45dd5027f4e3.png">

## This one 20, the other 16
<img width="78" alt="Screenshot 2023-03-03 at 7 04 07 PM" src="https://user-images.githubusercontent.com/1937582/222805427-6536b6bc-1014-4f05-86d3-35a4f9252de8.png">